### PR TITLE
fix(DB/creature_formations): Fixes Agro on pack

### PR DESCRIPTION
**Pull request by neifion-00000000**
## Changes Proposed:
-  Creature formations that should have been in #12371 implemented

## SOURCE:
Retail/Sniff

## How to Test the Changes:

1.  Check to see if all the snakes agro when 1 does, particularly when pulling from the side.  See #12371 for more information.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 1]
```diff
@@ -0,0 +1,96 @@
+--
+SET @GUID :=86939;
+DELETE FROM `creature_formations` WHERE `memberGUID` BETWEEN @GUID+0 AND @GUID+71;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES 
+(@GUID+0, @GUID+0, 0, 0, 3, 0, 0),
+(@GUID+0, @GUID+1, 0, 0, 3, 0, 0),
+(@GUID+0, @GUID+2, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:8]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+3, @GUID+3, 0, 0, 3, 0, 0),
+(@GUID+3, @GUID+4, 0, 0, 3, 0, 0),
+(@GUID+3, @GUID+5, 0, 0, 3, 0, 0),
+--
+(@GUID+6, @GUID+6, 0, 0, 3, 0, 0),
+(@GUID+6, @GUID+7, 0, 0, 3, 0, 0),
+(@GUID+6, @GUID+8, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:16]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+9, @GUID+9, 0, 0, 3, 0, 0),
+(@GUID+9, @GUID+10, 0, 0, 3, 0, 0),
+(@GUID+9, @GUID+11, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:20]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+12, @GUID+12, 0, 0, 3, 0, 0),
+(@GUID+12, @GUID+13, 0, 0, 3, 0, 0),
+(@GUID+12, @GUID+14, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:24]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+DELETE FROM `creature_formations` WHERE `memberGUID` BETWEEN @GUID+0 AND @GUID+71;
+INSERT INTO `creature_formations` (`leaderGUID`, `memberGUID`, `dist`, `angle`, `groupAI`, `point_1`, `point_2`) VALUES 
+(@GUID+0, @GUID+0, 0, 0, 3, 0, 0),
+(@GUID+0, @GUID+1, 0, 0, 3, 0, 0),
+(@GUID+0, @GUID+2, 0, 0, 3, 0, 0),
+--
+(@GUID+3, @GUID+3, 0, 0, 3, 0, 0),
+(@GUID+3, @GUID+4, 0, 0, 3, 0, 0),
+(@GUID+3, @GUID+5, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:12]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+9, @GUID+11, 0, 0, 3, 0, 0),
+--
+(@GUID+12, @GUID+12, 0, 0, 3, 0, 0),
+(@GUID+12, @GUID+13, 0, 0, 3, 0, 0),
+(@GUID+12, @GUID+14, 0, 0, 3, 0, 0),
+--
+(@GUID+15, @GUID+15, 0, 0, 3, 0, 0),
+(@GUID+15, @GUID+16, 0, 0, 3, 0, 0),
+(@GUID+15, @GUID+17, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:28]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+18, @GUID+18, 0, 0, 3, 0, 0),
+(@GUID+18, @GUID+19, 0, 0, 3, 0, 0),
+(@GUID+18, @GUID+20, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:32]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+21, @GUID+21, 0, 0, 3, 0, 0),
+(@GUID+21, @GUID+22, 0, 0, 3, 0, 0),
+(@GUID+21, @GUID+23, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:36]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+24, @GUID+24, 0, 0, 3, 0, 0),
+(@GUID+24, @GUID+25, 0, 0, 3, 0, 0),
+(@GUID+24, @GUID+26, 0, 0, 3, 0, 0),
+(@GUID+24, @GUID+27, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:41]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+28, @GUID+28, 0, 0, 3, 0, 0),
+(@GUID+28, @GUID+29, 0, 0, 3, 0, 0),
+(@GUID+28, @GUID+30, 0, 0, 3, 0, 0),
+(@GUID+28, @GUID+31, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:46]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 11]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+32, @GUID+32, 0, 0, 3, 0, 0),
+(@GUID+32, @GUID+33, 0, 0, 3, 0, 0),
+(@GUID+32, @GUID+34, 0, 0, 3, 0, 0),
+(@GUID+32, @GUID+35, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:51]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 12]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+36, @GUID+36, 0, 0, 3, 0, 0),
+(@GUID+36, @GUID+37, 0, 0, 3, 0, 0),
+(@GUID+36, @GUID+38, 0, 0, 3, 0, 0),
+(@GUID+36, @GUID+39, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:56]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 13]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+40, @GUID+40, 0, 0, 3, 0, 0),
+(@GUID+40, @GUID+41, 0, 0, 3, 0, 0),
+(@GUID+40, @GUID+42, 0, 0, 3, 0, 0),
+(@GUID+40, @GUID+43, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:61]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+44, @GUID+44, 0, 0, 3, 0, 0),
+(@GUID+44, @GUID+45, 0, 0, 3, 0, 0),
+(@GUID+44, @GUID+46, 0, 0, 3, 0, 0),
+(@GUID+44, @GUID+47, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:66]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 15]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+48, @GUID+48, 0, 0, 3, 0, 0),
+(@GUID+48, @GUID+49, 0, 0, 3, 0, 0),
+(@GUID+48, @GUID+50, 0, 0, 3, 0, 0),
+(@GUID+48, @GUID+51, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:71]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 16]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+52, @GUID+52, 0, 0, 3, 0, 0),
+(@GUID+52, @GUID+53, 0, 0, 3, 0, 0),
+(@GUID+52, @GUID+54, 0, 0, 3, 0, 0),
+(@GUID+52, @GUID+55, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:76]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 17]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+56, @GUID+56, 0, 0, 3, 0, 0),
+(@GUID+56, @GUID+57, 0, 0, 3, 0, 0),
+(@GUID+56, @GUID+58, 0, 0, 3, 0, 0),
+(@GUID+56, @GUID+59, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:81]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+60, @GUID+60, 0, 0, 3, 0, 0),
+(@GUID+60, @GUID+61, 0, 0, 3, 0, 0),
+(@GUID+60, @GUID+62, 0, 0, 3, 0, 0),
+(@GUID+60, @GUID+63, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:86]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 19]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+64, @GUID+64, 0, 0, 3, 0, 0),
+(@GUID+64, @GUID+65, 0, 0, 3, 0, 0),
+(@GUID+64, @GUID+66, 0, 0, 3, 0, 0),
+(@GUID+64, @GUID+67, 0, 0, 3, 0, 0),
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:91]: remove or make this a line break.

File: data/sql/updates/pending_db_world/rev_1658535848721793000.sql [hunk 20]
```diff
@@ -0,0 +1,96 @@
...
+(@GUID+68, @GUID+68, 0, 0, 3, 0, 0),
+(@GUID+68, @GUID+69, 0, 0, 3, 0, 0),
+(@GUID+68, @GUID+70, 0, 0, 3, 0, 0),
+(@GUID+68, @GUID+71, 0, 0, 3, 0, 0);
+--
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1658535848721793000.sql:96]: remove it and make it a blank line.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
Review left for adjustment to pass sql check

**Review by acidmanifesto — APPROVED** [review]
**Approve Review Noises**

**Comment by Gultask** [comment]
Saw no issues 👍

**Pull request merged**